cmd/kahoy/apply.go

```diff
@@ -259,6 +260,16 @@ func RunApply(ctx context.Context, cmdConfig CmdConfig, globalConfig GlobalConfi
 		}
 	}
 
+	// Wrap resource manager with timeout manager
+	manager, err = timeout.NewTimeoutManager(timeout.TimeoutManagerConfig{
```

Comment:
Would be handy that if the user sets the flag to `0` duration, it disables the timeout, so we could check here something similar to:

```go
if cmdConfig.Apply.ExecutionTimeout != 0 {
    manager, err := timeout...
}
```

Reply:
Right! Addressed in https://github.com/slok/kahoy/pull/168/commits/b7d0081a964c6abb71653173e1f99bdcb5367ebb